core/src/main/java/bio/terra/pearl/core/service/export/instance/ItemExportInfo.java

```diff
@@ -29,4 +33,10 @@ public class ItemExportInfo {
     private String questionText;
     @Builder.Default
     private List<QuestionChoice> choices = null;
+    /**
```

Comment:
The `ItemExportInfo` description says it is used for a survey question and "field of a data model". Just curious, what is the latter? 
The fields in this class seem biased toward question content. What is the advantage of sharing its use with something else? Do they share similar export processing?

Reply:
"field of a data model" would be something like "dateOfBirth" from a Profile object, or "complete" from SurveyResponse.
  I think you're right that this class could probably be usefully refactored into two classes that share an interface.